pkg/status/status.go

```diff
@@ -287,10 +287,10 @@ func getCommonStatus() (map[string]interface{}, error) {
 	stats["conf_file"] = config.Datadog.ConfigFileUsed()
 	stats["pid"] = os.Getpid()
 	stats["go_version"] = runtime.Version()
-	stats["agent_start"] = startTime.Format(timeFormat)
+	stats["agent_start_nano"] = startTime.UnixNano()
```

Comment:
I'm not sure it brings much benefit as this force us to call `formatUnixTime` with new parameter. If that's really helpful to keep sub-second precision, I think we should have `formatUnixTime` & `formatUnixTimeNano` to increase template readability.